myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -81,10 +82,14 @@
 
   public static final String DEFAULT_NATIVE_LIBRARY = "/usr/local/lib/libmesos.so";
 
-  public static final Integer DEFAULT_ZK_TIMEOUT = 20000;
+  public static final Integer DEFAULT_ZK_TIMEOUT    = 20000;
 
   public static final Integer DEFAULT_REST_API_PORT = 8192;
-
+  
+  public static final String DEFAULT_ROLE           = "*";
+  
+  public static final String DEFAULT_ZK_SERVERS     = "localhost:2181";
+  
```

Comment:
Let's not reformat to align = signs.  We're not doing it elsewhere so the code get's inconsistent.